radish34/contracts/src/utils.js

```diff
@@ -77,3 +77,114 @@ export const link = (bytecode, libraryName, libraryAddress) => {
   // swap out the placeholder with the library's deployed address:
   return bytecode.object.replace(pattern, address);
 };
+
+export const flattenDeep = arr => {
```

Comment:
Were these somewhere else before?

Reply:
yes these were in `/api/src/utils/crypto/conversions.js` these were meant to be used to convert hex to int. We can get rid of these from here. ideally, we should be making any such utils as part of the package

Reply:
I similarly had to duplicate some of the babyjubjub code within the `deploy/` package-- for now. Let's punt DRYing up utils to the package in the spirit of getting to stability and CI.

Reply:
I left them in but added a `TODO` comment explaining that they are duplicates and we intend to make them part of an importable package in the future.